dtrt-indent.el

```diff
@@ -408,6 +408,19 @@ variable."
                 :value-type (group variable))
   :group 'dtrt-indent)
 
+(defcustom dtrt-indent-run-after-smie nil
+  "*Non-nil means to run dtrt-indent even in modes using SMIE.
+
+Normally, dtrt-indent will detect SMIE-based modes and delegate
+to smie-config-guess. However, dtrt-indent configures some
```

Comment:
Please put Lisp symbols in old-fashioned single quotes, `smie-config-guess', and two spaces after periods (IOW, keep to Emacs "house style").